src/lib/assets/icons/download_list/easy.svg

```diff
@@ -1,33 +0,0 @@
-<svg width="136" height="74" viewBox="0 0 136 74" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Moved these to the static Synapse S3 assets bucket since they were not rendering properly when used in SWC.